doma-kotlin/src/main/kotlin/org/seasar/doma/kotlin/jdbc/criteria/expression/KExpressions.kt

```diff
@@ -200,13 +200,6 @@ object KExpressions {
         return Expressions.concat(left, right)
     }
 
-    fun <PROPERTY> concat(
```

Comment:
Could you please explain why this method needs to be removed?

Reply:
Sorry, I deleted it by mistake. I revert this removing.
And I fixed test-code to meet `concat` method test coverage.